tool/actions-plan-preview/main.go

```diff
@@ -152,7 +154,7 @@ func main() {
 		return
 	}
 
-	if bool(comment.IsMinimized) {
+	if comment.IsMinimized {
```

Comment:
I think this cast is to ensure we can notice if the lib type Boolean changed, so let's keep it.

Reply:
I see your point👍

Reply:
@khanhtc1202 
Applied!!
https://github.com/pipe-cd/pipecd/pull/4658/commits/449aa1b24cd33a627a0fd8909e20463294e9d8d2